internal/pipeline/aggregate.go

```diff
@@ -274,7 +281,7 @@ func (xfagg *aggregator) uploadFile(index int, agent upload.Agent, res *transfor
 
 	// Send Slack/PD or whatever notifications after the file is uploaded
 	if err := xfagg.notifyAfterUpload(filename, res.File, agent, err); err != nil {
-		xfagg.alertOnError(xfagg.logger.LogError(err).Err())
+		xfagg.alertOnWarning(xfagg.logger.LogError(err).Err())
 	}
```

Comment:
I'm not sure why this is changed to a warning. Failure to upload a file is a critical error.

Reply:
Personally I wasn't thinking of introducing a "warning" alert type. Slack could be configured instead of PagerDuty for non-production environments, or both.

Reply:
Oh no, that was a mistake.  I was looking at the comment above and updated the wrong place

Reply:
The issue mentioned having a lower level of alert notification, so that's why I added the "warning" alert type.  My changes don't take away the ability to do Slack and/or PagerDuty as an error level alert type for production or non-production.  The "warning" alert type gives additional optionality for different error levels, if wanted.

Reply:
Gotcha. I probably worded it poorly in the issue. Slack is typically a "lower priority" method of alerting someone as you can set work hours to snooze notifications. PagerDuty is an urgent/critical priority alerting method.